src/GameStatistics.ts

```diff
@@ -0,0 +1,74 @@
+import * as Unit from './entity/Unit';
+import * as Cards from './cards';
+
+//
+
+export interface IStatistics {
+  cardsCast: number;
+  myPlayerArrowsFired: number;
+  myPlayerDamageTaken: number;
+}
+
+// How can we initialize these without manually typing everything out 3+ times?
```

Comment:
Use can use `Object.assign({}, { cardsCast: 0, myPlayerArrowsFired: 0, myPlayerDamageTaken: 0 })` to clone the object.  (note it's a shallow clone but that should be fine since this object only has primitives and no nested objects.